node/service/src/benchmarking.rs

```diff
@@ -0,0 +1,430 @@
+			{
+				runtime::RuntimeCall::System(
+					runtime::SystemCall::remark { remark: vec![] }
+				)
+			},
+		}
+	}
+}
+
+/// Generates `Balances::TransferKeepAlive` extrinsics for the benchmarks.
+///
+/// Note: Should only be used for benchmarking.
+pub struct TransferKeepAliveBuilder {
+	client: Arc<FullClient>,
+	dest: AccountId,
+	chain: Chain,
+}
+
+impl TransferKeepAliveBuilder {
+	/// Creates a new [`Self`] from the given client and the arguments for the extrinsics.
+	pub fn new(client: Arc<FullClient>, dest: AccountId, chain: Chain) -> Self {
+		Self { client, dest, chain }
+	}
+}
+
+impl frame_benchmarking_cli::ExtrinsicBuilder for TransferKeepAliveBuilder {
+	fn pallet(&self) -> &str {
+		"balances"
+	}
+
+	fn extrinsic(&self) -> &str {
+		"transfer_keep_alive"
+	}
+
+	fn build(&self, nonce: u32) -> std::result::Result<OpaqueExtrinsic, &'static str> {
+		let signer = Sr25519Keyring::Bob.pair();
+		let period = polkadot_runtime_common::BlockHashCount::get()
+			.checked_next_power_of_two()
+			.map(|c| c / 2)
+			.unwrap_or(2) as u64;
+		let genesis = self.client.usage_info().chain.best_hash;
+		let current_block = 0;
+
+		identify_chain! {
+			self.chain,
+			nonce,
+			current_block,
+			period,
+			genesis,
+			signer,
+			{
+				runtime::RuntimeCall::Balances(runtime::BalancesCall::transfer_keep_alive {
+					dest: self.dest.clone().into(),
+					value: runtime::ExistentialDeposit::get(),
+				})
+			},
+		}
+	}
+}
+
+#[cfg(feature = "polkadot-native")]
+fn polkadot_sign_call(
+	call: polkadot_runtime::RuntimeCall,
+	nonce: u32,
+	current_block: u64,
+	period: u64,
+	genesis: H256,
+	acc: sp_core::sr25519::Pair,
+) -> OpaqueExtrinsic {
+	use polkadot_runtime as runtime;
+
+	let extra: runtime::SignedExtra = (
+		frame_system::CheckNonZeroSender::<runtime::Runtime>::new(),
+		frame_system::CheckSpecVersion::<runtime::Runtime>::new(),
+		frame_system::CheckTxVersion::<runtime::Runtime>::new(),
+		frame_system::CheckGenesis::<runtime::Runtime>::new(),
+		frame_system::CheckMortality::<runtime::Runtime>::from(sp_runtime::generic::Era::mortal(
+			period,
+			current_block,
+		)),
+		frame_system::CheckNonce::<runtime::Runtime>::from(nonce),
+		frame_system::CheckWeight::<runtime::Runtime>::new(),
+		pallet_transaction_payment::ChargeTransactionPayment::<runtime::Runtime>::from(0),
+		polkadot_runtime_common::claims::PrevalidateAttests::<runtime::Runtime>::new(),
+	);
+
+	let payload = runtime::SignedPayload::from_raw(
+		call.clone(),
+		extra.clone(),
+		(
+			(),
+			runtime::VERSION.spec_version,
+			runtime::VERSION.transaction_version,
+			genesis,
+			genesis,
+			(),
+			(),
+			(),
+			(),
+		),
+	);
+
+	let signature = payload.using_encoded(|p| acc.sign(p));
+	runtime::UncheckedExtrinsic::new_signed(
+		call,
+		sp_runtime::AccountId32::from(acc.public()).into(),
+		polkadot_core_primitives::Signature::Sr25519(signature.clone()),
+		extra,
+	)
+	.into()
+}
+
+#[cfg(feature = "westend-native")]
+fn westend_sign_call(
```

Comment:
nit: westend/kusama/rococo sign_call seems to be exact copies (mod runtime name). Maybe this could be somehow avoided (macro maybe?).